EventsExpress/Validation/EventValidator.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using EventsExpress.Db.Entities;
+using EventsExpress.Db.Enums;
```

Comment:
Do we use this someplace?

Reply:
No, it is used nowhere, I will delete this